src/main/g8/default.properties

```diff
@@ -1,3 +1,6 @@
-name=my-awesome-lambda
+name=Change Me
```

Comment:
`name=CHANGE_ME` rather than `name=Change Me`. Just makes it the same across the project so people can ctrl-f for all the places.